src/style.css

```diff
@@ -1,3 +1,150 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+:root {
+  --main-bg-color: #060b35;
+  --white-bg-color: #f3f3f3;
+  --blue-bg-color: #00acdc;
+  --silver-bg-color: #616070;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid var(--white-bg-color);
+}
+
+.card__image {
+  padding-top: 32px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  display: flex;
+  flex-direction: column;
+}
+
+.card__title {
+  color: var(--main-bg-color);
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 40px auto;
+  margin-bottom: 24px;
+  padding-left: 16px;
```

Comment:
just add paddings to the whole `.card`, dont add separate margins and paddings to different elements
fix it in the whole code